### PR TITLE
WINDUP-1252 - Support for Windup to run in server mode using RMI. Cle…

### PR DESCRIPTION
**Pull request by johnsteele**
…anup tooling-* dependencies.

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

**Comment by jsight** [comment]
retest

File: tooling/tests/src/test/java/org/jboss/windup/tooling/ExecutionBuilderTest.java [hunk 1]
```diff
@@ -178,23 +172,65 @@ public void testReloadGraph() throws IOException
...
     }
 
-    private Path getDefaultPath()
+    public static Path getDefaultPath()
     {
         return FileUtils.getTempDirectory().toPath().resolve("Windup").resolve("execbuildertest_" + RandomStringUtils.randomAlphanumeric(6));
     }
+    
+    @Test
+    public void testExecutionBuilderRegistered() throws Exception
```
Comment (jsight) [inline tooling/tests/src/test/java/org/jboss/windup/tooling/ExecutionBuilderTest.java]: Nicely done! :)

**Pull request merged**